sway-fmt-v2/src/constants.rs

```diff
@@ -14,9 +14,9 @@ pub const DEFAULT_TAB_SPACES: usize = 4;
 pub const DEFAULT_FN_CALL_WIDTH: usize = 60;
 /// Default max width of the args of a function-like attributes before falling back to vertical formatting.
 pub const DEFAULT_ATTR_FN_LIKE_WIDTH: usize = 70;
-/// Default max width in the body of a struct literal before falling back to vertical formatting.
+/// Default max width in the body of a structure literal before falling back to vertical formatting.
```

Comment:
Would you mind if we are explicit here and say `user-defined structure`? This is how I worded it in `Config` just forgot to update this

Reply:
Also the name of this `const` could be `DEFAULT_STRUCTURE_LIT_WIDTH`